conflation/main.py

```diff
@@ -28,13 +28,27 @@ def main():
         help="The number of processes to use to make requests, by default your # of cpus",
         default=multiprocessing.cpu_count(),
     )
+    arg_parser.add_argument(
+        "--valhalla_url",
+        type=str,
+        help="Base URL for an active Valhalla service",
+        required=True,
+    )
+    arg_parser.add_argument(
+        "--valhalla_headers",
+        type=str,
+        help="Additional http headers to send with the requests. Follows the http header spec, eg. some-header-name: some-header-value",
+        action="append",
+        nargs="*",
+    )
+
     # TODO: Change print() to use logger and add logging level as arg
 
     parsed_args = arg_parser.parse_args()
 
     # Create dirs
     bbox = parsed_args.bbox
-    output_dir, output_tmp_dir = util.initialize_dirs(bbox)
+    traces_dir, tmp_dir, map_matches_dir, results_dir = util.initialize_dirs(bbox)
```

Comment:
Bit of refactoring here, there are now different dirs for GPS traces, map match results, final result, etc.